demo/main.go

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
This file should be under cli/demo/ instead of just demo/

Reply:
ok